globalConfig.json

```diff
@@ -31,686 +31,749 @@
-            {
-              "field": "loglevel",
-              "label": "Log level",
-              "type": "singleSelect",
-              "options": {
-                "disableSearch": true,
-                "autoCompleteFields": [
-                  {
-                    "label": "DEBUG",
-                    "value": "DEBUG"
-                  },
-                  {
-                    "label": "INFO",
-                    "value": "INFO"
-                  },
-                  {
-                    "label": "WARNING",
-                    "value": "WARNING"
-                  },
+            "entity": [
+              {
+                "field": "name",
+                "label": "Account name",
+                "type": "text",
+                "required": true,
+                "help": "Enter a unique name for this account.",
+                "validators": [
                   {
-                    "label": "ERROR",
-                    "value": "ERROR"
+                    "type": "string",
+                    "minLength": 1,
+                    "maxLength": 25,
+                    "errorMsg": "Length of Account name should be between 1 and 25"
                   },
                   {
-                    "label": "CRITICAL",
-                    "value": "CRITICAL"
+                    "type": "regex",
+                    "pattern": "^[a-zA-Z]\\w*$",
+                    "errorMsg": "Account name must start with a letter and followed by alphabetic letters, digits or underscores."
                   }
                 ]
               },
-              "defaultValue": "INFO"
-            }
-          ]
-        },
-        {
-          "name": "proxy",
-          "title": "Proxy",
-          "entity": [
-            {
-              "field": "proxy_enabled",
-              "label": "Enable",
-              "type": "checkbox"
-            },
-            {
-              "type": "singleSelect",
-              "label": "Proxy Type",
-              "options": {
-                "disableSearch": true,
-                "autoCompleteFields": [
-                  {
-                    "value": "http",
-                    "label": "http"
+              {
+                "field": "url",
+                "label": "URL",
+                "type": "text",
+                "help": "Enter DataSet URL.",
+                "required": true,
+                "defaultValue": "https://app.scalyr.com",
+                "validators": [
+                  {
+                    "type": "regex",
+                    "pattern": "^https:\\/{2}(?:\\S)+\\.\\w{2,}$",
+                    "errorMsg": "URL must begin with https:// and end with top-level domain. E.g.: https://app.us1.dataset.com"
                   }
                 ]
               },
-              "defaultValue": "http",
-              "field": "proxy_type"
+              {
+                "field": "tenant",
+                "label": "Tenant",
+                "type": "text",
```

Comment:
Can we sue type `checkbox` here?

Reply:
According to the requirements it was made into the text box that supports True/ False or a blank. More over this ensures the backward compatibility